tidy3d/components/source.py

```diff
@@ -1120,6 +1135,18 @@ class PlaneWave(AngledFieldSource, PlanarSource):
         * `Using FDTD to Compute a Transmission Spectrum <https://www.flexcompute.com/fdtd101/Lecture-2-Using-FDTD-to-Compute-a-Transmission-Spectrum/>`__
     """
 
+    angular_spec: Union[FixedInPlaneKSpec, FixedAngleSpec] = pydantic.Field(
+        FixedInPlaneKSpec(),
+        title="Angular Dependence Specification",
+        description="Specification of plane wave propagation direction dependence on wavelength.",
```

Comment:
Could you provide an explanation of when a user should use `FixedInPlaneKSpec` or `FixedAngleSpec`? I guess usually when Bloch boundary conditions are used the source must be `FixedInPlaneKSpec`?

Reply:
added a short note about it. Also see new notebook https://github.com/flexcompute/tidy3d-notebooks/blob/daniil/fixed-angle/AngularFrequencyDependence.ipynb